src/classes/i18n.js

```diff
@@ -102,18 +102,33 @@ export default class i18n {
     /**
      * gets the translation for the key and replaces the variables
      * @public
+     * @params {string}    language
      * @params {string}    key
      * @params {object}    variables
      * @return {string}    translation
      */
-    trans (key, variables = {}) {
+    trans (lang, key, variables) {
+
+        // support for trans(key)
+        if (typeof key === 'undefined' && typeof variables === 'undefined') {
+            key = lang;
+            variables = {};
+            lang = this.lang;
+        }
+
+        // support for trans(key, variables)
+        if (typeof key === 'object') {
+            variables = key;
+            key = lang;
+            lang = this.lang;
+        }
```

Comment:
so probably check `this.lang` if it exists and throw an error if it is not yet set- if the user tries to use `trans`? I guess the chances of error occurring would be low if `default` is explicitly declared in the `.configuration()`?